tools/system_libs.py

```diff
@@ -41,7 +41,7 @@ def read_symbols(path, exclude=None):
       symbols = filter(lambda symbol: symbol not in exclude, symbols)
     return set(symbols)
 
-  default_opts = []
+  default_opts = ['-Werror']
```

Comment:
Hmm, this looks like a potentially-serious warning actually. The underlying issue, I believe, is that in asmjs long double is the same as double, but the wasm backend has decided otherwise. I'm not sure what the proper fix would be here. cc @sunfishcode